vignettes/advanced.Rmd

```diff
@@ -268,15 +466,161 @@ pool(anl_obj)
 
 # Delta Adjustment
 
+A delta-adjustment refers to a shift in the distribution of the (unobserved) outcome values by a certain amount. This might be used to perform sensitivity analyses.
```

Comment:
Would it be better to say "imputed" rather than "unobserved"  as the unobserved values never actually change, we just don't know them ?

Reply:
Also "might be" ?